concepticondata/concepticon.tsv

```diff
@@ -3148,3 +3148,33 @@ ID	GLOSS	SEMANTICFIELD	DEFINITION	ONTOLOGICAL_CATEGORY	REPLACEMENT_ID
 3152	MUNTJACS	Animals	Also known as barking deer, a kind of small deer of the genus Muntiacus.	Person/Thing	
 3153	LORD	Social and political relations	A man of noble rank or high office; a nobleman.	Person/Thing	
 3154	WILD BOAR	Animals	Also known as wild pig, an undomestic four-legged animal (Sus scrofa) living in the wild. 	Person/Thing	
+3155	DIBBLE STICK	Agriculture and vegetation	A piece of wood with a point used for making holes in the ground for seeds, seedlings or small bulbs.	Person/Thing	
+3156	GEBANG PALM	Agriculture and vegetation	Tree of the Corypha Utan species.	Person/Thing	
+3157	CIVET	Animals	A carnivorous mammal of the Viverridae family, found across South East Asia.	Person/Thing	
+3158	RICE EAR BUG	Animals	An insect of the species Leptocorisa oratoria, which gives of an unpleasant smell when disturbed and feeds on rice plants, destroying the crops.	Person/Thing	
+3159	BURDEN STICK	Basic actions and technology	A long piece of wood used to carry a burden.	Person/Thing	
+3160	SLAP	Basic actions and technology	Hit with the open palm of the hand	Action/Process	
+3161	CHEW BETEL	Food and drink	To consume a mixture of areca nut and betel leaf.	Action/Process	
+3162	CROSS-COUSIN	Kinship	The child of a parent’s opposite-sex sibling, for example the father’s sister’s son.	Person/Thing	
+3163	SQUAT	Motion	Have bent knees with the body weight is on the feet.	Action/Process	
+3164	ONE HUNDRED AND TWENTY THREE	Quantity	The cardinal number represented in Roman numerals as CXXIII, and in Arabic numerals as 123.	Other	
+3165	TWENTY THREE	Quantity	The cardinal number represented in Roman numerals as XXIII, and in Arabic numerals as 23.	Other	
+3166	TWENTY TWO	Quantity	The cardinal number represented in Roman numerals as XXII, and in Arabic numerals as 22.	Other	
+3167	SERMON	Religion and belief	A religious lecture or speech, addressing issues of religion or morality.	Person/Thing	
+3168	DOWRY	Social and political relations	Property, gifts or money transfered from the parents of a daughter to her new family at marriage.	Person/Thing	
+3169	TREATY	Social and political relations	Formal agreement between nations, states or other international bodies.	Person/Thing	
+3170	BATHE SOMEONE	The body	Clean someone else with water.	Action/Process	
+3171	RINGWORM	The body	Dermatophytosis, fungal skin infection making round patches on the skin.	Person/Thing	
+3172	SCABIES	The body	A contagious skin rash caused by the mite Sarcoptes scabiei.	Person/Thing	
+3173	TINEA	The body	Dermatophytosis, fungal skin infection making white spots/non-symmetrical white patches on the skin.	Person/Thing	
+3174	WAKE SOMEONE UP	The body	Rouse from sleep.	Action/Process	
+3175	MOUND OF STONES FOR RITUAL ACTIVITIES	The house	A mound of stones or altar around/on which ritual activities are held.	Person/Thing	
```

Comment:
is this concept THAT important tha tyou want to add it? maybe just leave it unlinked for the time being?

Reply:
I don't see how it's more strange than any of the other concepts we have, but sure, I can leave it unlinked. It's attested in about a quarter of our lects, and often with apparently mono- or bi-morphemic words (http://www.model-ling.eu/lexirumah/parameters/ritual_mound#8/-9.215/124.899), but we also have it in our cultural questionnaires with probably more explanation.

Reply:
Similar to a case we had above: if you have a cultural term (compare it with "mid-autumn festival") it is fine for me, but as a concrete cultural thing in one region, the gloss sounds too abstract.

Reply:
Ah, I get it. I'll have a look for a better gloss and maybe definition.